types/rascal/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/amqplib": "^0.5.13",
```

Comment:
@zijin-m You shouldn't need a package.json with an explicit dependency on `amqplib` -- it's on Definitely Typed, so you should be able to just import it.

There are sometimes exceptions when you need to rely on old version of a package, but your package.json specifies amqplib@0.5, which is the current version.

Reply:
Thank you very much, I have misunderstood the purpose of packeage.json